Library/EventTelemetry.ts

```diff
@@ -0,0 +1,19 @@
+import Telemetry = require("./Telemetry")
```

Comment:
What do you think of moving the Telemetry* classes into their own folder (either within Library or somewhere else)?

Reply:
Done.